src/libpriv/rpmostree-scripts.c

```diff
@@ -280,19 +280,26 @@ run_script_in_bwrap_container (int rootfs_fd,
 
   struct ChildSetupData data = { .stdin_fd = stdin_fd };
 
+  /* Only try to log to the journal if we're running in the system
+   * context; for unprivileged container builds we don't want to log
+   * to the journal, even if we have privileges to do so.
+   */
   const char *id = glnx_strjoina ("rpm-ostree(", pkg_script, ")");
-  data.stdout_fd = stdout_fd = sd_journal_stream_fd (id, LOG_INFO, 0);
-  if (stdout_fd < 0)
+  if (getuid () == 0)
```

Comment:
We also have to initialize the fds to `-1` in the `else` path, right?
Also, let's just move the `const char *id` to inside the if-statement?

Reply:
Yep, good catch.   As far as the `id`, I tried that originally but it's used in the error message below.